lib/decisive-engine-aws-cdk-stack.ts

```diff
@@ -236,8 +236,16 @@ export class DecisiveEngineAwsCdkStack extends cdk.Stack {
     });
     mdaiOperator.node.addDependency(otelOperator);
 
-    const mdaiOperatorCrManifest = yaml.load(readFileSync(path.join(__dirname, 'mdai-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
-    cluster.addManifest('mdaiOperatorCrManifest', mdaiOperatorCrManifest).node.addDependency(mdaiOperator);
+    const mdaiOperatorCrManifestYaml = yaml.load(readFileSync(path.join(__dirname, 'mdai-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
+    const mdaiOperatorCrManifest = cluster.addManifest('mdaiOperatorCrManifest', mdaiOperatorCrManifestYaml);
+    mdaiOperatorCrManifest.node.addDependency(mdaiOperator);
+
+    const otelOperatorCrManifestYaml = yaml.load(readFileSync(path.join(__dirname, 'otel-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
+    new eks.KubernetesManifest(this, 'otelOperatorCrManifest', {
+      cluster,
+      manifest: [otelOperatorCrManifestYaml],
+      overwrite: true,
+    }).node.addDependency(mdaiOperatorCrManifest);
```

Comment:
```suggestion
    const mdaiOperatorCrManifest = yaml.load(readFileSync(path.join(__dirname, 'mdai-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
    cluster.addManifest('mdaiOperatorCrManifest', mdaiOperatorCrManifest).node.addDependency(mdaiOperator);
```

operator applies annotations so applying the otel operator manifest is not needed.